saleor/order/migrations/tasks/saleor3_20.py

```diff
@@ -226,3 +226,35 @@ def set_order_subtotal_for_orders_created_from_bulk():
     if order_ids:
         _set_subtotal_for_orders_created_from_bulk(order_ids)
         set_order_subtotal_for_orders_created_from_bulk.delay()
+
+
+@app.task
+def fix_negative_total_net_for_orders_using_gift_cards_task(start_pk=0):
```

Comment:
It edge case scenario, but we can still end up with a order with negative total net in case when older worker create one and this task already ended processing all orders

Reply:
Yeah, wanted to note that but I agree this seems unnecessary to address.